src/components/jsonviewer/jsonviewer.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component, h, Prop } from '@stencil/core'
+
+@Component({
+  tag: 'json-viewer',
+  styleUrl: 'jsonviewer.scss',
+  shadow: true,
```

Comment:
The shadow property means that this web component is fully isolated from the rest of the doucments styling.  Nothing i put in this files css file affects the rest of the page, and this element is not affected by any other css in the page.